src/mps/mps.jl

```diff
@@ -310,9 +311,11 @@ function replacebond!(M::MPS,
   if ortho == "left"
     leftlim(M) == b-1 && setleftlim!(M, leftlim(M)+1)
     rightlim(M) == b+1 && setrightlim!(M, rightlim(M)+1)
+    normalize && (M[b+1] /= norm(M[b+1]))
```

Comment:
When I merge #314 you can change this to `M[b+1] ./= norm(M[b+1])` (a slight optimization).

Reply:
Great! Will do